styleguide/app/index.html

```diff
@@ -196,6 +207,78 @@ <h2 class="sidebar-content-heading">Components</h2>
                                 <div class="slider"></div>
```

Comment:
This "old" slider now has a permanently enabled tooltip that doesn't exist on the current version

![screenshot from 2017-02-21 17 39 49](https://cloud.githubusercontent.com/assets/1014341/23188479/d421d11e-f85c-11e6-9e00-5fa16b1ffdf0.png)